packages/react-components/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,422 @@
+            backgroundColor: 'red',
+            height: '50px',
+            width: '100px',
+          }}
+        ></div>
+        <div
+          onClick={() => dispatch({ type: 'reference2' })}
+          id="reference2"
+          style={{
+            display: 'block',
+            backgroundColor: 'red',
+            height: '50px',
+            width: '100px',
+            alignSelf: 'flex-start',
+          }}
+        ></div>
+
+        <div
+          onClick={() => dispatch({ type: 'reference3' })}
+          id="reference3"
+          style={{
+            display: 'block',
+            backgroundColor: 'red',
+            height: '50px',
+            width: '100px',
+          }}
+        ></div>
+
+        <TooltipUserGuideComponent
+          {...props}
+          isVisible={state.isVisible}
+          parentElementName={`#${state.reference}`}
+          zIndex={1000}
+          shouldSlide={true}
+          onClose={() => dispatch({ type: 'isVisible' })}
+        >
+          {state.reference === 'reference1' ? (
+            <UserGuideStep
+              header="Header - concise and clear"
+              image={{
+                src: beutifulImage,
+                alt: 'image',
+              }}
+              text="Tooltip content is used to explain the details of elements or features."
+              handleClickPrimary={() => dispatch({ type: 'reference2' })}
+              handleCloseClick={() => dispatch({ type: 'isVisible' })}
+              currentStep={1}
+              stepMax={3}
+              closeWithX
+            />
+          ) : null}
+
+          {state.reference === 'reference2' ? (
+            <UserGuideStep
+              header="Header - concise and clear"
+              image={{
+                src: beutifulImage,
+                alt: 'image',
+              }}
+              text="Tooltip content is used to explain the details of elements or features."
+              handleClickPrimary={() => dispatch({ type: 'reference3' })}
+              handleCloseClick={() => dispatch({ type: 'isVisible' })}
+              currentStep={2}
+              stepMax={3}
+              closeWithX
+            />
+          ) : null}
+
+          {state.reference === 'reference3' ? (
+            <UserGuideStep
+              header="Header - concise and clear"
+              image={{
+                src: beutifulImage,
+                alt: 'image',
+              }}
+              text="Tooltip content is used to explain the details of elements or features."
+              handleClickPrimary={() => dispatch({ type: 'isVisible' })}
+              handleCloseClick={() => {
+                dispatch({ type: 'isVisible' });
+              }}
+              currentStep={3}
+              stepMax={3}
+              closeWithX
+            />
+          ) : null}
+        </TooltipUserGuideComponent>
+      </div>
+    </div>
+  );
+};
+
+const TooltipInfoExample: React.FC<ITooltipProps> = (props) => {
+  return (
```

Comment:
```suggestion
const TooltipInfoExample: React.FC<ITooltipProps> = (props) => {
  return (
```
and here continuing the case of managed tooltip we can add some local state
```suggestion
const TooltipInfoExample: React.FC<ITooltipProps> = (props) => {
  const [isVisible, setIsVisible] = React.useState(false);
  return (
```